common/src/main/res/values/strings.xml

```diff
@@ -33,6 +36,16 @@
     <string name="pref_summary_port">The protobuf port of your hyperion instance</string>
     <integer name="pref_default_port">19445</integer>
 
+    <string name="pref_key_y_led">pref_key_y_led</string>
+    <string name="pref_title_y_led">Vertical LED Count</string>
+    <string name="pref_summary_y_led">The vertical count of LEDs on your hardware</string>
+    <integer name="pref_default_y_led">20</integer>
+
+    <string name="pref_key_x_led">pref_key_x_led</string>
+    <string name="pref_title_x_led">Horizontal LED Count</string>
+    <string name="pref_summary_x_led">The horizontal count of LEDs on your hardware</string>
+    <integer name="pref_default_x_led">36</integer>
```

Comment:
I feel this default is too specific. Not exactly sure where the 60x60 came from, but if we have no clue about the users's situation, let's guess on the high side and go with that. Or go with 60x34, which is 60 wide in a 16:9 aspect

Reply:
Sure we can go with whatever sounds the most reasonable. I just used the count from my setup. 60x34 sounds perfectly fine to me